### ISSUE TITLE
Repeat [for] [5] total occurences not translatable in Dutch

### ISSUE THREAD

**Issue by flipflop97** [dgsasha/remembrance#15]
I'm working on the Dutch translation for your app. The bottom setting of Repeat Options assumes an English sentence structure. This works perfect in English, but can become a problem in other languages.

![image](https://user-images.githubusercontent.com/13943260/223738497-75d376bb-2735-4512-9010-baa7b8618504.png)

In Dutch, the main problem is the sentence "Repeat [for] [5] total occurences". Which would be translated as "[5] keer herhalen" (literally "[5] times repeating"). If I try to force this sentence structure, it would become "Herhaal [5] keer" (literally "Repeat [5] times"). As you might see, the word [for] does not exist here. This becomes a problem, because the button would then become empty, or I have to force an incorrect/unnatural sentence.

This is just an example in Dutch, but I can imagine this is also a problem in other languages. It would be nice if this setting could be changed so it isn't a sentence, or that the whole sentence can be translated as one string, including the buttons.

EDIT: I think I've found a valid workaround for Dutch now, it's a totally different sentence but it works fine. Having a language-independant layout would probably be a better solution though, and could benefit other translators too.

**Comment by flipflop97** [comment]
Btw, a totally different thing: Would you like the app name translated? I couldn't find a note in Weblate about that.

**Comment by dgsasha** [comment]
![Screenshot from 2023-03-08 12-07-01](https://user-images.githubusercontent.com/86973360/223782226-2d7afe31-4532-4110-ac25-8d810540a5b4.png)
How does this look? I thought about this maybe being an issue recently, so thanks for bringing it to my attention.
Also, I do want the app name translated.

**Comment by dgsasha** [comment]
>  I'll have to move part of the sentence from the button to the end (after times)

Yeah for now just do that, later I will probably change the interface so the entries aren't a part of the sentence. It would make the dialog less compact but it probably would be better that way.

Reopening for now.
